mobileapp/README.md

```diff
@@ -1,40 +1,54 @@
-Requires docker build (from project folder / parent repository)
-```
-docker-compose build public_web 
-```
+# Mobileapp
+
+##
+
+A few prerequisite:
+
+- iOS: install cocoapods
+- Run `npm install -g cordova@9.0.0 code-push-cli@2.1.9`
+- Run `npm install`
+- Optional (required for some internal Tomorrow builds): download `GoogleService-Info.plist` from Firebase
+
+If you want your local JavaScript changes to be reflected, you need to disable Codepush by commenting out the `codePush.sync` calls in `../web/src/cordova.js`.
+
+To build the JavaScript:
 
-To build js:
 ```
-./build.sh
+docker-compose build web && ./build.sh
```

Comment:
Might be good to document the need to expose the `build-arg` in order to get proper access to the API.